src/Purchases.ts

```diff
@@ -0,0 +1,51 @@
+import { Platform } from 'react-native';
+import type { Double } from 'react-native/Libraries/Types/CodegenTypes';
+
+export interface ChargebeeConfig {
+  site: string;
+  publishableApiKey: string;
+  androidSdkKey: string;
+  iOsSdkKey: string;
+}
+
+export interface Product {
+  readonly id: string;
+  readonly title: string;
+  readonly price: string;
+  readonly currencyCode: string;
+}
+
+export interface Purchase {
+  readonly subscriptionId: string;
+  readonly planId: string;
+  readonly status: string;
+}
+
+export interface Subscription {
+  readonly activatedAt: Double | null;
```

Comment:
all of these properties can be `null`? 🤔

Reply:
Since this object should follow Chargebee [Subscriptions](https://apidocs.chargebee.com/docs/api/subscriptions?prod_cat_ver=2#subscription_attributes), some of these are always present.

However in the iOS [model](https://github.com/chargebee/chargebee-ios/blob/acc1c25209bbc70673f7db2f4b0fd57249c66e75/Chargebee/Classes/Subscription/CBSubscription.swift#L37) We have marked all of them as optional, while in the Android [model](https://github.com/chargebee/chargebee-android/blob/c92478c636a7eef47374eb544fc58552679d7b3f/chargebee/src/main/java/com/chargebee/android/models/SubscriptionDetail.kt#L3) we follow the CB model. 
So had marked all as optional to make them common. Let me see if I can unwrap the optionals from within iOS and throw an error otherwise.

Reply:
@cb-sriramthiagarajan On further discussion with the team, found the non-optional fields and have updated them here.